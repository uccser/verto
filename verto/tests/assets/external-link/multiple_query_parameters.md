Check out this [website](https://www.google.com?key1=value1&key2=value2).
