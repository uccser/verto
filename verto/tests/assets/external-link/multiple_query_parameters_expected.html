<p>Check out this <a href="https://www.google.com?key1=value1&amp;key2=value2" target="_blank">website</a>.</p>
